lib/manageiq/integration/foreman_manager.rb

```diff
@@ -0,0 +1,188 @@
+require "fileutils"
+
+# A wrapper around the foreman utility for running ManageIQ in the integration
+# environment, with either the server acting more "dev like" or "prod like"
+# depending on the scenario (local development versus CI).
+#
+module ManageIQ
+  module Integration
+    class ForemanManager
+      def self.pid_file
+        @pid_file ||= Environment.tmp_dir.join('foreman.pid')
```

Comment:
Note that we have a PidFile class that can be used in lib.  Might be better to enhance that than recreate anything.

Reply:
Sorry, saw this yesterday and was working on API performance stuff and forgot to respond, but yes I agree 100%, just didn't realize there was a `PidFile` class/module, and so I will look into seeing about using that.

Reply:
@Fryguy This should be implemented now, in the second to last commit (at the time of writing), but I might move some of the changes that I added to that file into a separate PR.

That said, wouldn't mind some feedback if you think _**how**_ I did that makes sense.